tests/test_simple.rs

```diff
@@ -19,6 +20,21 @@ mod test {
             nodes[i].write().unwrap().start();
         }
 
+        // waiting for nodes to be synced
+        let now = Instant::now();
+        loop {
+            if Instant::now() - now > Duration::from_secs(10) {
+                panic!("nodes are not synced in 10s");
+            }
+            let all_synced = nodes
+                .iter()
+                .all(|node| node.read().unwrap().view_account(&account_names[0]).is_ok());
+            if all_synced {
+                break;
+            }
+            std::thread::sleep(Duration::from_secs(1));
```

Comment:
0.1 instead of 1, given in the test env the syncing is probably fast, and some tests take on the order of 1s total?